docs/src/modeler/parsing.md

```diff
@@ -1,9 +1,11 @@
 # Constructing a System from raw data
 
 ## Supported Formats
-* PowerSystems table data
-* MATPOWER (parsed by PowerModels)
-* PSS/E (parsed by PowerModels)
+- PowerSystems table data (CSV Files)
+- MATPOWER (parsed by PowerModels)
```

Comment:
"develped" by PowerModels, or "borrowed from". Technically it's not parsed by PM since we don't use PM here.